package.json

```diff
@@ -15,20 +15,17 @@
     "web-vitals": "^3.4.0"
   },
   "devDependencies": {
-    "@testing-library/jest-dom": "^6.1.2",
+    "@testing-library/jest-dom": "^6.1.3",
     "@testing-library/react": "^14.0.0",
     "@testing-library/user-event": "^14.4.3",
     "@types/jest": "^29.5.4",
-    "@types/node": "^20.5.4",
+    "@types/node": "^20.5.9",
     "@types/react": "^18.2.21",
     "@types/react-dom": "^18.2.7",
     "@wixc3/react-board": "^2.3.0",
     "react-scripts": "^5.0.1",
     "sass": "^1.66.1",
-    "typescript": "~5.1.6"
-  },
-  "overrides": {
-    "typescript": "~5.1.6"
+    "typescript": "4.9.5"
```

Comment:
Good, I guess upgrading typescript here - https://github.com/codux-demos/ducktube/commit/7cb1e57185196011657fd7201ec720233edf7265 was unnecessary and we, not clear why, followed.